src/prefect/agent/agent.py

```diff
@@ -45,9 +45,8 @@ def __init__(self) -> None:
         self._verify_token(token)
 
         logger = logging.getLogger("agent")
```

Comment:
Not sure if it matters but this is not a Prefect logger

Reply:
That's intentional - these logs are intended to be streamed to stdout in whatever platform the agent is running in.  We _should_ introduce the notion of "Agent logs" to store in Cloud, and find a way of exposing them in a way that is flow-independent but that's a future feature.